actionpack/test/dispatch/prefix_generation_test.rb

```diff
@@ -22,6 +22,23 @@ def klass.name; self end
     def to_model; self; end
     def persisted?; true; end
   end
+  class Author
```

Comment:
I think we always have a linebreak between classes:
```suggestion

  class Author
```